server/controllers/availability.js

```diff
@@ -0,0 +1,67 @@
+const asyncHandler = require('express-async-handler');
+const Availability = require('../models/Availability');
+
+// @route GET /availability/?monday[from]={900}&monday[to]={1300}, etc
+// @desc Get available sitter list based on required customer availability from query
+// @access Private
+exports.getAvailableSitters = asyncHandler(async (req, res, next) => {
+  const query = { ...req.query };
+
+  const queryHasAvailabilities = query.hasOwnProperty('monday')
+    || query.hasOwnProperty('tuesday')
+    || query.hasOwnProperty('wednesday')
+    || query.hasOwnProperty('thursday')
+    || query.hasOwnProperty('friday');
```

Comment:
Interesting - let's also add saturday and sunday

Reply:
done